extensions/amp-next-page/0.1/config.js

```diff
@@ -95,13 +90,23 @@ function assertSelectors(selectors) {
 /**
  * @param {!Element} context
  * @param {*} reco
- * @param {string} origin
- * @param {string=} sourceOrigin
+ * @param {string} documentUrl
  */
-function assertReco(context, reco, origin, sourceOrigin) {
+function assertReco(context, reco, documentUrl) {
+  user().assertString(reco.ampUrl, 'ampUrl must be a string');
+
+  // Rewrite relative URLs to absolute, relative to the canonical URL.
+  const isRelativeUrl = !/^(?:[a-z]+:)?\/\//.test(reco.ampUrl);
```

Comment:
no need to do the check, it is safe to call `resolveRelativeUrl` regardless. It is no-op if Url is already absolute.

Reply:
Done.